pyam/compute.py

```diff
@@ -249,6 +251,248 @@ def bias(self, name, method, axis):
         """
         _compute_bias(self._df, name, method, axis)
 
+    def kaya_variables(self, scenarios, append=False):
+        """Compute the variables needed to compute Kaya factors
+        for the Kaya Decomposition Analysis.
```

Comment:
The first sentence of the docstring should fit in one line, per numpy conventions.
```suggestion
        """Compute the variables needed to compute Kaya factors
```